portfolio/src/main/webapp/script.js

```diff
@@ -57,39 +63,90 @@ function addProjectDescription(project) {
   wrapper.style.backgroundColor = projectNameToData[projectName]['backgroundColor']; 
 }
 
-
 /**
  * This is the helper function for addProjectDescription.
- * This function creates a data key-value pair object for each project.
- * Input: 
+ * This function creates a data key-value pair object for each project. 
  */
 function createProjectData(projectDetail, backgroundColor) {
- 
   projectData = {}
   projectData['projectDetail'] = projectDetail;
   projectData['backgroundColor'] = backgroundColor; 
-
   return projectData; 
 }
 
 /**
  * This function enables the highlighting of a selected section from the navigation bar. 
  * This function is called by the clicking of any one of the boxes in the navigation bar.
  * Input: section to select
- * Effect: selected section will have their border shown, and all the other sections will not have the border. 
+ * Effect: selected section will have their border shown, and all the other sections 
+ *         will not have the border. 
  */
 function highlightSection(selectedIdString) {
-
   document.getElementById(selectedIdString).style.border = 'solid';
   document.getElementById(selectedIdString).style.borderColor = '#c5cf0a';
 
   const navbarSections = document.getElementsByClassName('navbarSection');
 
-  Object.keys(navbarSections).forEach(
-    (key) => {
-      const section = navbarSections[key];
+  Object.values(navbarSections).forEach(
+    (section) => {
       if (section.id !== selectedIdString) {
         section.style.border = 'none'; 
       }
     }); 
-}
\ No newline at end of file
+}
+
+/**
+ * This function fetches the text from /data, with additional parameters
+ * added to the request's URL string. 
+ * Then, the function puts the received text into the quoteWrapper
+ * that gets displayed on the UI.  
+ */
+function getQuotes() {
+  const mainURL = '/data';
+  const numToDisplayURL = '?numToDisplay=';
+  const numToDisplay = document.getElementById("numToDisplay").value; 
```

Comment:
Let's use single quotes for string literals across this file.

Reference:
https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

Reply:
Done.